api/geth_backend.go

```diff
@@ -1442,6 +1442,8 @@ func (b *GethStatusBackend) prepareSettings(request *requests.CreateAccount, mne
 		s.Fleet = &request.WakuV2Fleet
 	}
 
+	s.ThirdpartyServicesEnabled = request.ThirdpartyServicesEnabled
```

Comment:
We could add it to `defaultSettings` and then you don't need to pass it from the client.

Reply:
I need to pass it from client because the user flow is such that the user has the option to turn off third party services on onboarding itself, and we should pass it in that case right?